tests/integration/plugins/test_get_old_tvars.py

```diff
@@ -0,0 +1,132 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from cylc.flow.scheduler_cli import get_option_parser
+from cylc.flow.parsec.exceptions import Jinja2Error
+from cylc.flow.pre_configure.get_old_tvars import main as get_old_tvars
+import pytest
+from pytest import param
+from types import SimpleNamespace
+
+from cylc.flow.scripts.validate import (
+    _main as validate,
+    get_option_parser as validate_gop
+)
+from cylc.flow.scripts.view import (
+    _main as view,
+    get_option_parser as view_gop
+)
+from cylc.flow.scripts.graph import (
+    _main as graph,
+    get_option_parser as graph_gop
+)
+from cylc.flow.scripts.config import (
+    _main as config,
+    get_option_parser as config_gop
+)
+
+
+@pytest.fixture(scope='module')
+def create_workflow(mod_one_conf, mod_flow, mod_scheduler):
```

Comment:
Fixtures should generally be nouns unless they return callables, in which case verbs make sense.

E.G. `test_workflow`, `example_workflow`.

Reply:
Done